web/core/components/exporter/export-modal.tsx

```diff
@@ -35,21 +35,26 @@ export const Exporter: React.FC<Props> = observer((props) => {
   const { workspaceSlug } = useParams();
   // store hooks
   const { workspaceProjectIds, getProjectById } = useProject();
+  const { projectsWithCreatePermissions } = useUser();
 
-  const options = workspaceProjectIds?.map((projectId) => {
-    const projectDetails = getProjectById(projectId);
+  const options = projectsWithCreatePermissions
```

Comment:
Can we write it like this?
`wsProjectsWithCreatePermisisons = _.intersection(workspaceProjectIds,projectsWithCreatePermissions)`
This is more readable.